### PR TITLE
Add some more structure, reorganizes some resources

### PR DESCRIPTION
**Pull request by epersonae**
@Bradamant3 @barriebyron I've got some rearranged general content and links in this PR, along with a general guideline for how to organize the page.

### CODE REVIEW

**Comment by epersonae** [comment]
Also fixes issue #1500

File: docs/guide/writing/style-guides.md [hunk 1]
```diff
@@ -6,35 +6,72 @@ A style guide saves documentarians time and trouble by providing a single refere
...
+## Sample writing guides
+
 Classics for software documentation include:
 
 - [Microsoft Writing Style Guide](https://docs.microsoft.com/en-us/style-guide/welcome/)
 - [IBM style guide](https://www.ibm.com/developerworks/library/styleguidelines/)
 - [Apple Style Guide](https://help.apple.com/applestyleguide/)
 
-Others you might find useful:
+### More style guides from commercial organizations
```
Comment (barriebyron) [inline docs/guide/writing/style-guides.md]:
```suggestion
### More enterprise software style guides
```

Comment (plaindocs) [reply docs/guide/writing/style-guides.md]: :+1:

File: docs/guide/writing/style-guides.md [hunk 2]
```diff
@@ -6,35 +6,72 @@ A style guide saves documentarians time and trouble by providing a single refere
 The guidelines in a style guide help writers to produce documentation that has the same tone and grammatical style, regardless of who writes the documentation.
 A consistent tone and style can make your content easier to read by reducing your users' [cognitive load](https://en.wikipedia.org/wiki/Cognitive_load) and increasing their confidence in the content's authority. 
 
+## Overall structure of this page (WIP)
```
Comment (plaindocs) [inline docs/guide/writing/style-guides.md]: Maybe remove the WIP (it is implied throughout)

**Review by plaindocs — COMMENTED** [review]
This is great, happy to merge this in after the two suggested fixes. Thank you! :-)

File: docs/guide/writing/style-guides.md (hunk 2 above)
Comment (plaindocs) [reply docs/guide/writing/style-guides.md]:
```suggestion
## Structure of this page
```

**Review by plaindocs — APPROVED** [review]
Merging! Thanks. :-)

**Pull request merged**